### PR TITLE
Updated form/* articles to Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: form/form_collections.rst [hunk 1]
```diff
@@ -510,7 +510,7 @@ you will learn about next!).
 
         .. code-block:: yaml
 
-            # src/Resources/config/doctrine/Task.orm.yml
+            # src/Resources/config/doctrine/Task.orm.yaml
```
Comment (yceruto) [inline form/form_collections.rst]: I'm not sure if `yaml` ext is already supported for Doctrine :/ https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Mapping/Driver/SimplifiedYamlDriver.php#L33

File: form/create_form_type_extension.rst [hunk 2]
```diff
@@ -233,7 +236,7 @@ Specifically, you need to override the ``file_widget`` block:
 
     .. code-block:: html+php
 
-        <!-- app/Resources/file_widget.html.php -->
+        <!-- templates/form/file_widget.html.php -->
```
[the remark this answers is not in the document]
Comment (javiereguiluz) [reply form/create_form_type_extension.rst]: I'm not sure ... but I've reverted my changes. In any case, we'll remove all PHP template examples very soon.

File: form/form_collections.rst (hunk 1 above)
Comment (javiereguiluz) [reply form/form_collections.rst]: That's sad. Reverted.

**Comment by javiereguiluz** [comment]
@yceruto thanks for your review!

**Pull request merged**